spec/_attachments/ic.did

```diff
@@ -131,18 +133,36 @@ type bitcoin_get_balance_args = record {
     min_confirmations : opt nat32;
 };
 
+type bitcoin_get_balance_result = satoshi;
+
 type bitcoin_get_balance_query_args = record {
     address : bitcoin_address;
     network : bitcoin_network;
     min_confirmations : opt nat32;
 };
 
+type bitcoin_get_balance_query_result = satoshi;
+
+type bitcoin_get_current_fee_percentiles_args = record {
+    network : bitcoin_network;
+};
+
+type bitcoin_get_current_fee_percentiles_result = vec millisatoshi_per_byte;
+
 type bitcoin_send_transaction_args = record {
     transaction : blob;
     network : bitcoin_network;
 };
 
-type millisatoshi_per_byte = nat64;
+type bitcoin_get_block_headers_args = record {
+    start_height : bitcoin_block_height;
+    end_height : opt bitcoin_block_height;
+};
+
+type bitcoin_get_block_headers_result = record {
+    tip_height : bitcoin_block_height;
+    block_headers : vec bitcoin_block_header;
```

Comment:
I'm wondering if it makes sense to wrap each `block_header` in a record. That way, we'd leave the door open in the future to add additional to each block header (e.g. number of confirmations).

Reply:
I don't have a strong opinion about this.
Since we also return a record, an alternative would be to add more information to that record, right?
For example, if we add the `start_height` and `end_height`, then the number of confirmations for each block header is clear (given the `tip_height`). What do you think?

Reply:
SGTM, I also don't have a strong opinion about this, and as you say we can always add additional information by adding new fields to this record.